UI/window-basic-main.hpp

```diff
@@ -37,6 +37,7 @@
 #include "window-missing-files.hpp"
 #include "window-projector.hpp"
 #include "window-basic-about.hpp"
+#include "window-whats-new.hpp"
```

Comment:
I don‘t think the What‘s New window is part of OBSBasic‘s class identity or even API (external or internal), so I don’t think it needs to be included in the header nor does it need to be class member.

The pointer can live in an anonymous namespace in the implementation and the header just needs to be included there.

Because what this effectively does is make everything that uses something from `OBSBasic` (which unfortunately is almost everything) also pull in the new dialog class even though nobody but the main window will (or should) use it.

Reply:
As seen below this also creates a circular header include which is only ‘fixed’ by `#pragma once`, but also a good indicator that the relationship between the classes is not correct.

Reply:
+1 to this.

 (sorry accidentally resolved while trying to comment).

Reply:
Good point, fixed - though not sure if that's the best spot for it. Re-tested and still works correctly.